torch/nn/modules/pooling.py

```diff
@@ -45,6 +45,10 @@ class MaxPool1d(_MaxPoolNd):
     for :attr:`padding` number of points. :attr:`dilation` is the stride between the elements within the
     sliding window. This `link`_ has a nice visualization of the pooling parameters.
 
+    Note:
+        When ceil_mode=True, sliding windows are allowed to go off-bounds as long as it starts within the
```

Comment:
"as long as it..." -> "if they start within the left padding or the input. Windows that would start in the right padded region are ignored."